README.md

```diff
@@ -21,16 +21,13 @@ The current MVP covers the most used technologies which are:
 
 More languages/tools will come in the future.
 
-## Usage
+## QuickStart
 
-- Install yeoman and the generator: `yarn global add yo @thetribe/generator-project`
-- Create a new directory for your project and go to it
+You can quickly use the generator by following the [quickstart guide](doc/QUICKSTART.md)
 
-### Guided mode
+## Advanced Usage
 
-Run `yo @thetribe/project` and follow the prompts to generate your project
-
-### Manual mode
+You can also run the generator manually which let you pick what you individual template you want to add 
```

Comment:
Missed this typo in the previous review :slightly_smiling_face: 
```suggestion
You can also run the generator manually which let you pick what individual template you want to add 
```

Reply:
🤦 there should be a coding principle acronym for Read Yourself Twice Little Padawan